### PR TITLE
[Update] How to Install NGINX

### PR DESCRIPTION
**Pull request by hzoppetti**
guides for basic nginx installation
- Debian 10
- CentOS 8
- Ubuntu 18.04 LTS

(apologies for typo in branch name)

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 6565e33a7c4e370f74a0d5fc1c6110b7acbc1f3a

https://deploy-preview-3139--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/web-servers/nginx/how-to-install-nginx-ubuntu-18-04/index.md [hunk 1]
```diff
@@ -21,44 +23,71 @@ external_resources:
...
-## Configure NGINX
+## Add a Basic Site
 
-### Add Basic Site
+1.  Create a new directory for your site.
 
-NGINX site-specific configuration files are kept in `/etc/nginx/sites-available` and symlinked into  `/etc/nginx/sites-enabled/`. Generally you will want to create a separate original file in the `sites-available` directory for each domain or subdomain you will be hosting, and then set up a symlink in the `sites-enabled` directory.
+        sudo mkdir /var/www/example.com
+
+1.  NGINX site-specific configuration files are kept in `/etc/nginx/sites-available` and symlinked to  `/etc/nginx/sites-enabled/`. Generally, you will create a new file containing a [*server block*](https://www.nginx.com/resources/wiki/start/topics/examples/server_blocks/) in the `sites-available` directory for each domain or subdomain you will be hosting. Then, you will set up a symlink to your files in the `sites-enabled` directory.
```
Comment (andystevensname) [inline docs/web-servers/nginx/how-to-install-nginx-ubuntu-18-04/index.md]: I think this paragraph needs to be about the location of site files, as this paragraph is (rightly) included in the section below it.

File: docs/web-servers/nginx/how-to-install-nginx-centos-8/index.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+## Install NGINX
+
+Currently, the best way to install NGINX on CentOS 8 is to use the version included in CentOS's repositories:
+
+    sudo yum update
+    sudo dnf install @nginx
+
+## Add a Basic Site
+
+1.  Create a new directory for your site.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/web-servers/nginx/how-to-install-nginx-centos-8/index.md]: Added

File: docs/web-servers/nginx/how-to-install-nginx-debian-10/index.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+## Install NGINX
+
+Currently, the best way to install NGINX on Debian 10 is to use the version included in Debian's repositories:
+
+    sudo apt update
+    sudo apt install nginx
+
+## Add a Basic Site
+
+1.  Create a new directory for your site.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/web-servers/nginx/how-to-install-nginx-debian-10/index.md]: Added

File: docs/web-servers/nginx/how-to-install-nginx-ubuntu-18-04/index.md [hunk 4]
```diff
@@ -21,44 +23,71 @@ external_resources:
...
+Currently, the best way to install NGINX on Ubuntu 18.04 LTS is to use the version included in Ubuntu's repositories:
 
     sudo apt update
     sudo apt install nginx
 
-## Configure NGINX
+## Add a Basic Site
 
-### Add Basic Site
+1.  Create a new directory for your site.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/web-servers/nginx/how-to-install-nginx-ubuntu-18-04/index.md]: Added

File: docs/web-servers/nginx/how-to-install-nginx-ubuntu-18-04/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/web-servers/nginx/how-to-install-nginx-ubuntu-18-04/index.md]: Fixed

**Pull request merged**